language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java

```diff
@@ -136,7 +138,7 @@ public static void printSyntax(PrintStream out, List<Token> tokens) {
   /**
    * Connects to the Natural Language API using Application Default Credentials.
    */
-  public static CloudNaturalLanguageAPI getLanguageService() 
+  public static CloudNaturalLanguageAPI getLanguageService()
     throws IOException, GeneralSecurityException {
     GoogleCredential credential =
         GoogleCredential.getApplicationDefault().createScoped(CloudNaturalLanguageAPIScopes.all());
```

Comment:
Done. I didn't see additional tokens in syntax, I currently have lemma, text, part of speech, dependency edge. I did, however, add "mentions" to entities, which was what I think you meant.